test/acceptance/behave/environment.py

```diff
@@ -125,6 +125,20 @@ def init_feature():
     scenarios_status = {"untested": 0, "skipped": 0, "passed": 0, "failed": 0}
 
 
+def __add_chars(text, before, after, char_to_add=" "):
+    """
+    append chars before or/and after in a text
+    :param text: text to modify
+    :param before: number of blanks added to the beginning of the text
```

Comment:
Yes, initially in this function only blank spaces are appended, now, it is used with any character.
Fixed in  	b5998b6